src/app/cat-card/cat-card.component.html

```diff
@@ -0,0 +1,20 @@
+<mat-card class="example-card" appearance="outlined">
+  <mat-card-header>
+    <div mat-card-avatar class="example-header-image"></div>
+    <mat-card-title>{{ cat.name }}</mat-card-title>
+    <mat-card-subtitle>{{ cat.breed }}</mat-card-subtitle>
+  </mat-card-header>
+  <div class="img-container">
+    <img mat-card-image [src]="cat.image.src" [alt]="cat.image.alt" />
+  </div>
+  <mat-card-content>
+    <p>
+      {{ cat.content }}
+    </p>
+  </mat-card-content>
+  <mat-card-actions>
+    <ng-container *ngFor="let btnName of cat.actions">
+      <button mat-button>{{ btnName }}</button>
```

Comment:
I don't see this variable in cat object
could you fix it with cat.actions array?